doc/src/build/wallet.md

```diff
@@ -498,7 +498,7 @@ Here is an example:
 {"contents":{"fields":{"id":{"fields":{"id":{"fields":{"id":{"fields":{"bytes":"eea4167be074537f4a2879c7781d8ef4ffd651cc"},"type":"0x2::ID::ID"}},"type":"0x2::ID::UniqueID"},"version":0},"type":"0x2::ID::VersionedID"},"value":100000},"type":"0x2::Coin::Coin<0x2::SUI::SUI>"},"owner":{"AddressOwner":[102,175,56,152,231,85,139,121,225,21,171,97,24,74,149,132,151,209,144,90]},"tx_digest":[0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0]}
 ```
 
-## Transferring objects
+## Transferring coins
 
 If you inspect a newly created account, you would expect the account does not own any object. Let us inspect the fresh account we create in the [Generating a new account](#generating-a-new-account) section (`C72CF3ADCC4D11C03079CEF2C8992AEA5268677A`):
```

Comment:
I think there are a lot of interchangeable uses of `object` where `coin` could be used in this section. However to make it easier I think if you just add a line at the beginning of this section that describes the following it will clear everything up. 

`Coins ARE objects but have a specific use case that allow for native commands like transfer-coin/merge-coin/split-coin to be used. This is different from non-coin objects that can only be mutated via Move calls.`

Also a warning or linking to the section for non-coin transfers via Move would be useful at the top here. I will refer to your judgment on whats most clear for the reader.

Reply:
Done all around. Thanks!